public/javascripts/SVValidate/mobile/mobileValidate.css

```diff
@@ -276,4 +276,4 @@ html {
     height: 90px;
     width: 730px;
     z-index: 1;
-}
+}
```

Comment:
Looks like you accidentally removed the newline at the end of the file. Could you reverse that? It might be a setting in your IDE, and you should change that setting to be the opposite, we want a newline at the end of every file :+1: